src/overlays/actors/ovl_En_In/z_en_in.c

```diff
@@ -21,92 +44,1623 @@ const ActorInit En_In_InitVars = {
+
+extern AnimationHeader D_06015918;
+
+extern AnimationHeader D_0601C0B0;
+
+extern AnimationHeader D_0601A140;
+
+extern AnimationHeader D_0601B904;
+
+extern AnimationHeader D_0601B3C4;
+
+extern AnimationHeader D_06019EB4;
+
+extern Gfx D_060137A0[];
+
+extern Gfx D_06014420[];
+
+extern Gfx D_06014040[];
+
+extern Gfx D_06013670[];
+
+extern Gfx D_06014AE0[];
+
+extern Gfx D_06012A78[];
+
+extern Gfx D_06013DE0[];
+
+extern Gfx D_060138D0[];
+
+extern Gfx D_06013540[];
+
+extern Gfx D_06012DF8[];
+
+extern Gfx D_060145D8[];
+
+extern Gfx D_06014860[];
+
+extern Gfx D_06014710[];
+
+extern Gfx D_06013A00[];
+
+extern Gfx D_06013F10[];
+
+extern Gfx D_06014C30[];
+
+extern Gfx D_060149A8[];
+
+extern Gfx D_06013440[];
+
+static ColliderCylinderInit sCylinderInit = {
+    {
+        COLTYPE_HIT0,
+        AT_NONE,
+        AC_ON | AC_TYPE_PLAYER,
+        OC1_ON | OC1_TYPE_ALL,
+        OC2_TYPE_1,
+        COLSHAPE_CYLINDER,
+    },
+    {
+        ELEMTYPE_UNK1,
+        { 0x00000000, 0x00, 0x00 },
+        { 0xF7CFFFFF, 0x00, 0x00 },
+        TOUCH_NONE | TOUCH_SFX_NORMAL,
+        BUMP_ON,
+        OCELEM_ON,
+    },
+    { 18, 64, 0, { 0, 0, 0 } },
+};
+
+static ColliderJntSphElementInit sJntSphElementsInit[1] = {
+    {
+        {
+            ELEMTYPE_UNK0,
+            { 0xF7CFFFFF, 0x00, 0x00 },
+            { 0x00000000, 0x00, 0x00 },
+            TOUCH_ON | TOUCH_SFX_NORMAL,
+            BUMP_NONE,
+            OCELEM_NONE,
+        },
+        { 12, { { 1600, 0, 0 }, 5 }, 200 },
+    },
+};
+
+static ColliderJntSphInit sJntSphInit = {
+    {
+        COLTYPE_NONE,
+        AT_ON | AT_TYPE_ENEMY,
+        AC_NONE,
+        OC1_NONE,
+        OC2_TYPE_2,
+        COLSHAPE_JNTSPH,
+    },
+    1,
+    sJntSphElementsInit,
+};
+
+CollisionCheckInfoInit2 sColChkInfoInit2 = {
+    0, 0, 0, 0, MASS_HEAVY,
+};
+
+static DamageTable sDamageTable = {
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+};
+
+static ActorAnimationEntryS sAnimations[] = {
+    { &D_06001D10, 1.0f, 0, -1, 0, 0 },  { &D_06001D10, 1.0f, 0, -1, 0, -4 }, { &D_06014F8C, 1.0f, 0, -1, 0, 0 },
+    { &D_06014F8C, 1.0f, 0, -1, 0, -4 }, { &D_06000CB0, 1.0f, 0, -1, 0, -4 }, { &D_060003B4, 1.0f, 0, -1, 0, -4 },
+    { &D_06001BE0, 1.0f, 0, -1, 0, -4 }, { &D_06015918, 1.0f, 0, -1, 0, -4 }, { &D_0601C0B0, 1.0f, 0, -1, 0, 0 },
+    { &D_0601C0B0, 1.0f, 0, -1, 0, -4 }, { &D_0601A140, 1.0f, 0, -1, 0, 0 },  { &D_0601A140, 1.0f, 0, -1, 0, -4 },
+    { &D_0601B904, 1.0f, 0, -1, 0, 0 },  { &D_0601B904, 1.0f, 0, -1, 0, -4 }, { &D_0601B3C4, 1.0f, 0, -1, 0, 0 },
+    { &D_0601B3C4, 0.0f, 0, -1, 2, 0 },  { &D_0601B3C4, 1.0f, 0, -1, 0, -4 }, { &D_06019EB4, 1.0f, 0, -1, 2, -4 },
+};
+
+static UNK_TYPE D_808F6C0C[] = {
+    0x0FA00004, 0x00010003, 0x17700004, 0x00010006, 0x0FA00004, 0x00010003, 0x17700004, 0x00010006,
+};
```

Comment:
```suggestion
static s16 D_808F6C0C[] = {
4000, 4, 1, 3, 6000, 4, 1, 6, 
4000, 4, 1, 3, 6000, 4, 1, 6,
};
```
If I had to guess, the data is probably broken up into s16. Worth checking to make sure it still matches. It's okay if you want to leave it though until there's more info

Reply:
It does match and it does make the most sense as `s16`. I'm going to make the change.